src/html5/parser.rs

```diff
@@ -140,6 +140,18 @@ impl ActiveElement {
     }
 }
 
+pub struct Html5ParserOptions {
+    pub scripting_enabled: bool,
+}
```

Comment:
See my comments above about stutter — this might look better as `html5::ParserOptions`.